### PR TITLE
Add Event Listeners

### PR DESCRIPTION
**Pull request by mrparkersson**
Pull request
Amo Parker Prince

What does this PR do?

Add event listeners to buttons and elements

Description of the task to be completed
Adding clicks in Javascript file

Add event listeners to close the menu and open it 

clone the repository
git clone https://github.com/mrparkersson/mobile-version.git
navigate to the folder
cd portfolio-skeleton
checkout the feature
javascript-create-my-page

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+document.querySelector(".union").addEventListener("click", () => {
+  document.querySelector(".hamburger").style.display = "flex";
+  // document.body.style.overflow = "hidden";
```
Comment (carlos-ssh) [inline script.js:3]: # 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- ⏏︎ Remove commented code from your project.

File: index.css [hunk 2]
```diff
@@ -3,16 +3,133 @@
   margin: 0;
```
Comment (carlos-ssh) [inline index.css:3]: - Enable ESLint in your project and check the javascript linters.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @mrparkersson , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  e.addEventListener("click", () => {
+    document.querySelector(".hamburger").style.display = "none";
+  });
+});
+
+window.onresize = () => {
+  if (window.innerWidth > 992) {
+    document.querySelector(".hamburger").style.display = "none";
+  }
+};
```
Comment (descholar-ceo) [inline script.js:19]: - [ ] As the previous reviewer stated it, you were required to setup eslint for github actions in order to track eslint issues, kindly edit `.github/workflows/linters.yml` file to include eslint rules. If you follow this lesson, you can find a way out: https://github.com/microverseinc/linters-config/tree/master/html-css-js

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @mrparkersson, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @mrparkersson , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**